srcdeps-core/src/main/java/org/l2x6/srcdeps/core/util/SrcdepsCoreUtils.java

```diff
@@ -91,6 +96,64 @@ public FileVisitResult visitFileFailed(Path file, IOException exc) throws IOExce
     }
 
     /**
+     * Makes sure that the given directory exists. Tries creating {@link #CREATE_RETRY_COUNT} times.
+     *
+     * @param dir
+     *            the directory {@link Path} to check
+     * @throws IOException if the directory could not be created or accessed
+     */
+    public static void ensureDirectoryExists(Path dir) throws IOException {
+        IOException toThrow = null;
+        for (int i = 0; i < CREATE_RETRY_COUNT; i++) {
+            try {
+                Files.createDirectories(dir);
+                if (Files.exists(dir)) {
+                    return;
+                }
+            } catch (AccessDeniedException e) {
+                /* Workaround for https://bugs.openjdk.java.net/browse/JDK-8029608 */
+                try {
+                    Thread.sleep(10);
+                } catch (InterruptedException e1) {
```

Comment:
Thread.sleep is not affected by spurious wakeups IIRC, so I suppose someone really wants us to stop at this point.
Maybe we should log an exception and 
Thread.currentThread().interrupt(); 
in the catch block